sklearn/linear_model/coordinate_descent.py

```diff
@@ -611,8 +627,10 @@ class ElasticNet(LinearModel, RegressorMixin):
         independent term in decision function.
 
     n_iter_ : array-like, shape (n_targets,)
-        number of iterations run by the coordinate descent solver to reach
-        the specified tolerance.
+        For ``solver='cd'``, the number of iterations run by the solver to
+        reach the specified tolerance.
+        For ``solver='saga'``, the number of full passes on all samples until
+        convergence.
```

Comment:
I realize that the `dual_gap_` attribute is not mentioned here. This is an oversight. This should be fixed.

Reply:
It never has been. I can add it in this PR or open a new issue. What do you prefer?